### DISCUSSION TITLE
transcription quality of STT differs between headphones and mobile phones

### DISCUSSION THREAD

**Discussion by nweiner** [deepgram/community#1015] [General help] [answered]
Has anyone encountered this issue? The transcription quality of speech-to-text differs between headphones and mobile phones. Especially when using noise-cancelling headphones, the transcription quality is poor, with many short words not being captured or transcribed incorrectly. However, when speaking directly into the phone, the transcription effect is much better. Is Deepgram aware of this situation? Is there any way to improve the transcription quality when using headphones for calls?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @nweiner, my assumption would be that for some reason, you are passing lower-quality audio to Deepgram via the noise-canceling headphones. Our transcription will be higher-accuracy with clearer audio. If you can share Deepgram request IDs with me, I can see if we still have the audio available in our temporary debugging storage for you to review.

**Reply by nweiner** [reply]
Thanks jkroll, We have already devised a solution for it.

**Reply by Mahesh1356** [reply]
What was the solution?
